src/errors/types.js

```diff
@@ -87,26 +87,27 @@ export const RUNTIME_ERRORS = {
     browserNotSet:                                      'E1007',
     testFilesNotFound:                                  'E1008',
     noTestsToRun:                                       'E1009',
-    cannotFindReporterForAlias:                         'E1010',
-    multipleSameStreamReporters:                        'E1011',
-    optionValueIsNotValidRegExp:                        'E1012',
-    optionValueIsNotValidKeyValue:                      'E1013',
-    invalidSpeedValue:                                  'E1014',
-    invalidConcurrencyFactor:                           'E1015',
-    cannotDivideRemotesCountByConcurrency:              'E1016',
-    portsOptionRequiresTwoNumbers:                      'E1017',
-    portIsNotFree:                                      'E1018',
-    invalidHostname:                                    'E1019',
-    cannotFindSpecifiedTestSource:                      'E1020',
-    clientFunctionCodeIsNotAFunction:                   'E1021',
-    selectorInitializedWithWrongType:                   'E1022',
-    clientFunctionCannotResolveTestRun:                 'E1023',
-    regeneratorInClientFunctionCode:                    'E1024',
-    invalidClientFunctionTestRunBinding:                'E1025',
-    invalidValueType:                                   'E1026',
-    unsupportedUrlProtocol:                             'E1027',
-    testControllerProxyCannotResolveTestRun:            'E1028',
-    timeLimitedPromiseTimeoutExpired:                   'E1029',
+    noTestsToRunDueFiltering:                           'E1010',
```

Comment:
the `TestRunInfo` object in the `test-run-done` actions contains the `errs[]` property. If we modify codes of existings errors, it can lead to problems for users.